test/python/test_fusedbatchnorm.py

```diff
@@ -0,0 +1,76 @@
+# ==============================================================================
+#  Copyright 2018 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow FusedBatchNorm test
+
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+from common import NgraphTest
+
+
+class TestFusedBatchNorm(NgraphTest):
+  x = np.random.rand(64, 3, 10, 8).astype('f')
+  scale = [1.0, 0.9, 1.1]
+  offset = [0.1, 0.2, -.3]
+
+  def test_fusedbatchnorm_nchw(self):
+    with self.device:
+      norm = tf.nn.fused_batch_norm(self.x, self.scale, self.offset,
+                                    data_format='NCHW')
+
```

Comment:
The TranslateOp function of batchnorm has 2 paths one for training one for testing.
By default the 'is_training' parameter of tf.nn.fused_batch_norm is true. We should have a testcase that activates the test path of the TranslateOp with is_training=false

Reply:
Not related to this change directly: tf_is_training in TranslateFusedBatchNormGradOp is being set, but never used. Maybe we can get rid of that?

Reply:
"is_training attribute not present, setting to true" ... Not sure if we prefer true or false. For grad ops we can clearly chose is_trainable to be true, not sure for non-grad ops.

Reply:
> "is_training attribute not present, setting to true" ... Not sure if we prefer true or false. For grad ops we can clearly chose is_trainable to be true, not sure for non-grad ops.

I think I wrote that, back when I wasn't sure whether we would need to set the default attribute values ourselves in the builder, or could rely on them to be inserted by TF itself. It appears to be the latter.

Same thing with the `epsilon` attribute.